src/components/ha-climate-control.html

```diff
@@ -0,0 +1,104 @@
+<link rel='import' href='../../bower_components/polymer/polymer.html'>
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+
+
+<dom-module id="ha-climate-control">
+  <template>
+    <style is="custom-style" include="iron-flex iron-flex-alignment"></style>
+    <style>
+      :host {
+        display: flex;
+      }
+      /* local DOM styles go here */
+      .container-ha-climate-control {
```

Comment:
You can remove this whole class and just merge all these properties into `:host`. And then you can also remove `display: flex` (as `@apply(--layout-flex)` takes care of that including vendor prefix)